content/about/notes/1.1/index.md

```diff
@@ -1,32 +1,149 @@
 ---
 title: Istio 1.1
-publishdate: 2019-03-01
+publishdate: 2019-03-20
 icon: notes
 ---
 
-TODO announcement
+We're proud to release Istio 1.1! We have spent the last 8 months doing some significant improvements to the overall
+product, with fixes & features from Google, IBM, VMware, Huawei, RedHat, Cisco, SAP, Salesforce, Pivotal, SUSE, Datadog
+and Lightstep, to name a few. Special thanks to all of our end-users for providing feedback, feature requests and
+testing the release candidates at various scales.
+
+These release notes describe what's different between Istio 1.0 and Istio 1.1.
```

Comment:
should we say 1.0.0 or 1.0.6 here?  There is also a bunch of changes in fix packs.

Reply:
I switched to 1.0.6